graphql/config.js

```diff
@@ -24,4 +24,5 @@ module.exports = {
   SENDGRID_API_KEY: process.env.SENDGRID_API_KEY,
   TRUEX_APPLICATION_SECRET: process.env.TRUEX_APPLICATION_SECRET,
   E2E_MISSIONS_TEST_TAB_GOAL: process.env.E2E_MISSIONS_TEST_TAB_GOAL,
+  GROWTHBOOK_ENV: process.env.GROWTHBOOK_ENV,
```

Comment:
FYI, our current setup doesn't load env vars to the AWS Lambda function (we could and probably should do this) but rather writes them to this file.